index.js

```diff
@@ -35,30 +61,29 @@ app.get('/posts', async (req, res) => {
 	}
 });
 
-app.get('/users/:id', async (req, res) => {
+app.get('/posts/:id', async (req, res) => {
 	const { id } = req.params;
-	console.log(id);
 	try {
-		const user = await userDB.get(id);
-		res.status(200).json(user);
+		const post = await postDB.get(id);
+		res.status(200).json(post);
 	}
 	catch(err) {
 		res.status(500).json({ message: 'Error getting the data'});
 	}
 });
 
-app.get('/users/:id/posts', async (req, res) => {
-	const { id } = req.params;
-	console.log(id);
+app.post('/users', upperUser, async (req,res) => {
+	const { name } = req.body;
 	try {
-		const posts = await userDB.getUserPosts(id);
-		res.status(200).json(posts);
+		const count = await userDB.insert({name});
+		res.status(201).json({message: 'User successfully added'});
```

Comment:
Nothing major. You just want to have stuff like this `({message: 'User successfully added'})` as `({ message: 'User successfully added' })`. Like how you had it for line 76 `const { name } = req.body;`